Python/2. variables.md

```diff
@@ -46,7 +53,9 @@ Dictionary: {"what" : 3, "blah" : "oh no"}
 List: [3, 4, "D", 2.4, True, False, [[3, 4, 5, 6, 7], True]]
 ```
 
-Note: `\n` means a new line. If you add that in the middle of a print statement in double or single quotes, it will print the rest on a new line.
+`\n` means a new line. If you add that in the middle of a print statement in double or single quotes, it will print the rest on a new line. You can also add newlines inside of lists, dictionaries, and tuples `()`. To check the type of a variable, enter `>>> type(aVar)` or `print(type(aVar))`.
+
+_Note: Because the parameters for a function are inside a tuple, you can add newlines like in the code above. This shouldn't affect the actual code execution._
```

Comment:
wdym

Reply:
BTW this is for the Note

Reply:
The parenthesese `()` next to a function accepts parameters. It's actually a tuple...

Reply:
I didn't know that...cool

Reply:
yeah @whippingdot, @BD103 is right, here is an example:
``` python
def puts(text):
    print(text)
puts("Hello World")
puts
# the second puts will do nothing
```
I think I may have explained it wrong...

Reply:
hey I just added the note as I had forgotten earlier